esp/services/ws_workunits/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ target_link_libraries ( ws_workunits
          eclrtl
          deftype
          workunit
+         dfuwu
```

Comment:
This is unrelated but I can't build on OSX without it. Could be pulled to a separate PR if Gavin is being picky.